build_tools/torchscript_e2e_heavydep_tests/facebook_dlrm.py

```diff
@@ -0,0 +1,839 @@
+import torch
+import torch_mlir
+import numpy as np
+import torch.nn as nn
+from typing import Dict, List
+from numpy import random as ra
+from torch.utils.data import Dataset
+
+from torch_mlir_e2e_test.torchscript.framework import TestUtils
+from torch_mlir_e2e_test.torchscript.registry import register_test_case
+from torch_mlir_e2e_test.torchscript.annotations import annotate_args, export
+
+class DLRM_Net(nn.Module):
+    
+    emb_l_q: List[torch.Tensor]
+    def create_mlp(self, ln, sigmoid_layer):
+
+        layers = nn.ModuleList()
+        for i in range(0, ln.size - 1):
+            n = ln[i]
+            m = ln[i + 1]
+
+            # construct fully connected operator
+            LL = nn.Linear(int(n), int(m), bias=True)
+
+            mean = 0.0  
+            std_dev = np.sqrt(2 / (m + n))
+            W = np.random.normal(mean, std_dev, size=(m, n)).astype(np.float32)
+            std_dev = np.sqrt(1 / m)  
+            bt = np.random.normal(mean, std_dev, size=m).astype(np.float32)
+
+            LL.weight.data = torch.tensor(W, requires_grad=True)
+            LL.bias.data = torch.tensor(bt, requires_grad=True)
+            layers.append(LL)
+
+            # construct sigmoid or relu operator
+            if i == sigmoid_layer:
+                layers.append(nn.Sigmoid())
+            else:
+                layers.append(nn.ReLU())
+
+        return torch.nn.Sequential(*layers)
+
+    def create_emb(self, m, ln, weighted_pooling=None):
+        emb_l = nn.ModuleList()
+        v_W_l = []
+        for i in range(0, ln.size):
+
+            n = ln[i]
+            n = n.item()
+
+            if self.qr_flag and n > self.qr_threshold:
+                EE = QREmbeddingBag(
```

Comment:
where is this imported from?

Reply:
actually just realized that code isn't even triggered while tracing... so maybe I should remove that or add another test case that triggers that control flow?
its imported from here in the original file: from tricks.qr_embedding_bag import QREmbeddingBag